[2022-11-17 23:43:20,574] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: hello_dag.call_bye 2022-11-16T23:42:42.789245+00:00 [queued]>
[2022-11-17 23:43:20,640] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: hello_dag.call_bye 2022-11-16T23:42:42.789245+00:00 [queued]>
[2022-11-17 23:43:20,643] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-11-17 23:43:20,651] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2022-11-17 23:43:20,653] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-11-17 23:43:20,693] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): call_bye> on 2022-11-16T23:42:42.789245+00:00
[2022-11-17 23:43:20,722] {standard_task_runner.py:52} INFO - Started process 95 to run task
[2022-11-17 23:43:20,754] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'hello_dag', 'call_bye', '2022-11-16T23:42:42.789245+00:00', '--job-id', '348', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/index.py', '--cfg-path', '/tmp/tmpybf5josc', '--error-file', '/tmp/tmp57itkth5']
[2022-11-17 23:43:20,760] {standard_task_runner.py:77} INFO - Job 348: Subtask call_bye
[2022-11-17 23:43:21,014] {logging_mixin.py:104} INFO - Running <TaskInstance: hello_dag.call_bye 2022-11-16T23:42:42.789245+00:00 [running]> on host 1d9aa7be56f4
[2022-11-17 23:43:21,198] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=hello_dag
AIRFLOW_CTX_TASK_ID=call_bye
AIRFLOW_CTX_EXECUTION_DATE=2022-11-16T23:42:42.789245+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-16T23:42:42.789245+00:00
[2022-11-17 23:43:21,218] {python.py:118} INFO - Done. Returned value was: bye everyone
[2022-11-17 23:43:21,337] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=hello_dag, task_id=call_bye, execution_date=20221116T234242, start_date=20221117T234320, end_date=20221117T234321
[2022-11-17 23:43:21,450] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-11-17 23:43:21,510] {local_task_job.py:146} INFO - Task exited with return code 0
